metadata-db-app/src/cmr/metadata_db/migrations/077_remove_email_column.clj

```diff
@@ -0,0 +1,16 @@
+(ns cmr.metadata-db.migrations.077-remove-email-column
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(defn up
+  "Migrates the database up to version 77"
+  []
+  (println "cmr.metadata-db.migrations.077-drop email-column up...")
+  (h/sql "alter table METADATA_DB.cmr_subscriptions drop column EMAIL_ADDRESS"))
+
+
```

Comment:
Couple of whitespace nitpicks. Extra whitespace here, and needs a newline at the end of the file. Looks good otherwise

Reply:
done.